spk/homeassistant/Makefile

```diff
@@ -1,13 +1,14 @@
 SPK_NAME = homeassistant
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 1
+include ../../cross/$(SPK_NAME)/version
```

Comment:
I find it overkill compare to simply copy-paste of version string used in all packages...